aviary/subsystems/propulsion/engine_deck.py

```diff
@@ -55,7 +55,7 @@
 SHAFT_POWER_CORRECTED = EngineModelVariables.SHAFT_POWER_CORRECTED
 RAM_DRAG = EngineModelVariables.RAM_DRAG
 FUEL_FLOW = EngineModelVariables.FUEL_FLOW
-ELECTRIC_POWER_IN = EngineModelVariables.ELECTRIC_POWER_IN
+ELECTRIC_POWER = EngineModelVariables.ELECTRIC_POWER_IN
```

Comment:
is this intentional?

Reply:
Calling attention to this edit again. I recall that there was an earlier PR that changed the names and definitions of these variables. I have some questions about the new ones, but this might be a merge mistake.

Reply:
I've reverted it back, should be good now